schema/definitions/0.8.0/examples/table_wellpicks.yml

```diff
@@ -0,0 +1,122 @@
+# Example metadata for an inplace volume table.
+
+$schema: https://main-fmu-schemas-dev.radix.equinor.com/schemas/0.8.0/fmu_results.json
+version: "0.8.0"
+source: fmu
+
+tracklog:
+  - datetime: 2020-10-28T14:28:02
+    user:
+      id: peesv
+    event: created
+  - datetime: 2020-10-28T14:46:14
+    user:
+      id: peesv
+    event: updated
+
+class: table # class is the main identifier of the data type.
+
+fmu: # the fmu-block contains information directly related to the FMU context
+  model:
+    name: ff
+    revision: 21.0.0.dev
+    description:
+      - detailed description
+      - optional
+
+  workflow:
+    reference: rms/import
+
+  context:
+    stage: realization
+
+  case:
+    name: MyCaseName
+    uuid: 8bb56d60-8758-481a-89a4-6bac8561d38e
+    user:
+      id: jriv # $USER from ERT
+    description:
+      - yet other detailed description
+      - optional
+
+  iteration:
+    id: 0 # always an int, will be 0 for e.g. "pred"
+    uuid: 4b939310-34b1-4179-802c-49460bc0f799 # hash of case.uuid + iteration.id
+    name: "iter-0" # /"pred"
+    restart_from: 15ce3b84-766f-4c93-9050-b154861f9100   # fmu.iteration.uuid for another iteration
+
+  realization:
+    id: 33
+    uuid: 29a15b21-ce13-471b-9a4a-0f791552aa51
+    name: "realization-33"
+    parameters: # directly pass parameters.txt. This is potentially a lot of content, only a stub is included here.
+      SENSNAME: faultseal
+      SENSCASE: low
+      RMS_SEED: 1006
+      INIT_FILES:
+        PERM_FLUVCHAN_E1_NORM: 0.748433
+        PERM_FLUVCHAN_E21_NORM: 0.782068
+      KVKH_CHANNEL: 0.6
+      KVKH_US: 0.6
+      FAULT_SEAL_SCALING: 0.1
+      FWL_CENTRAL: 1677
+
+file:
+  relative_path: realization-33/iter-0/share/results/tables/wellpicks.csv # case-relative
+  absolute_path: /some/absolute/path/realization-33/iter-0/share/results/tables/wellpicks.csv
+  checksum_md5: kjhsdfvsdlfk23knerknvk23  # checksum of the file, not the data.
+  size_bytes: 5010144
+
+data: # The data block describes the actual data (e.g. surface). Only present in data objects
+
+  content: well_picks   # white-listed and standardized
+
+  name: inplace_mygrid
+  stratigraphic: false
+  format: csv
+  vertical_domain: depth # / time / null
+  depth_reference: msl # / seabed / etc
+  spec: # class/layout dependent, optional? Can spec be expanded to work for all data types?
+    ncol: 281
+    nrow: 441
+    undef: -999.25  # Allow both number and string
+    columns:
+      - BULK_OIL
+      - NET_OIL
+      - PORE_OIL
```

Comment:
Not columns you would expect for well picks

Reply:
Yes, good catch, thanks. Ctrl+c/Ctrl+v error... Sorted out now 👍